### PR TITLE
os/bluestore: Add missing __func__ in dout

### PR DESCRIPTION
**Pull request by lixiaoy1**
With __func__ it makes users more easily to track logs.

Signed-off-by: Xiaoyan Li xiaoyan.li@intel.com

### CODE REVIEW

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -4220,7 +4220,7 @@ int BlueStore::_open_fm(bool create)
         }
         assert(start + l + u <= end);
 
-	dout(20) << "  free 0x" << std::hex << start << "~" << l
+	dout(20) << __func__ << "  free 0x" << std::hex << start << "~" << l
```
Comment (joscollin) [inline src/os/bluestore/BlueStore.cc]: Nit: there is an unwanted space (2 spaces) before `free 0x`. Can you fix it along with these changes ?

File: src/os/bluestore/BlueStore.cc [hunk 2]
```diff
@@ -7918,7 +7919,7 @@ void BlueStore::_txc_write_nodes(TransContext *txc, KeyValueDB::Transaction t)
       extent_part = p.get_logical_offset() - onode_part - blob_part;
     }
 
-    dout(20) << "  onode " << o->oid << " is " << bl.length()
+    dout(20) << __func__  << "  onode " << o->oid << " is " << bl.length()
```
Comment (joscollin) [inline src/os/bluestore/BlueStore.cc]: ditto

File: src/os/bluestore/BlueStore.cc [hunk 3]
```diff
@@ -7945,13 +7946,15 @@ void BlueStore::_txc_write_nodes(TransContext *txc, KeyValueDB::Transaction t)
     auto sbid = sb->get_sbid();
     get_shared_blob_key(sbid, &key);
     if (sb->persistent->empty()) {
-      dout(20) << "  shared_blob 0x" << std::hex << sbid << std::dec
+      dout(20) << __func__ << "  shared_blob 0x"
```
Comment (joscollin) [inline src/os/bluestore/BlueStore.cc]: ditto

File: src/os/bluestore/BlueStore.cc [hunk 4]
```diff
@@ -7945,13 +7946,15 @@ void BlueStore::_txc_write_nodes(TransContext *txc, KeyValueDB::Transaction t)
...
+               << std::hex << sbid << std::dec
 	       << " is empty" << dendl;
       t->rmkey(PREFIX_SHARED_BLOB, key);
     } else {
       bufferlist bl;
       ::encode(*(sb->persistent), bl);
-      dout(20) << "  shared_blob 0x" << std::hex << sbid << std::dec
+      dout(20) << __func__ << "  shared_blob 0x"
```
Comment (joscollin) [inline src/os/bluestore/BlueStore.cc]: ditto

File: src/os/bluestore/BlueStore.cc (hunk 1 above)
Comment (lixiaoy1) [reply src/os/bluestore/BlueStore.cc]: will update.

File: src/os/bluestore/BlueStore.cc (hunk 2 above)
Comment (lixiaoy1) [reply src/os/bluestore/BlueStore.cc]: yes.

File: src/os/bluestore/BlueStore.cc (hunk 3 above)
Comment (lixiaoy1) [reply src/os/bluestore/BlueStore.cc]: yes

File: src/os/bluestore/BlueStore.cc (hunk 4 above)
Comment (lixiaoy1) [reply src/os/bluestore/BlueStore.cc]: yes.

**Comment by lixiaoy1** [comment]
@joscollin Thanks for the review. Patch is updated.

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-09-24_17:52:02-rados-wip-sage-testing-2017-09-24-1125-distro-basic-smithi/

**Pull request merged**